### DISCUSSION TITLE
A PB-QSP Model of the incretin hormones GLP-1 and GIP and a DPP4-inhibitor sitagliptin

### DISCUSSION THREAD

**Discussion by PavelBal** [Open-Systems-Pharmacology/Forum#733] [Announcement]
Dear all,

we are happy to share a PB-QSP model of secretion and elimination of the incretin hormones GLP-1 and GIP and the DPP4-inhibitor sitagliptin. The model has been developed in MoBi and can be integrated into the published [QSP Glucose-Insulin-Model](https://github.com/Open-Systems-Pharmacology/Glucose-Insulin-Model).

The model can be downloaded from [its repository](https://github.com/Open-Systems-Pharmacology/Incretins-Model), the respective publication is 

Balazki, P., Schaller, S., Eissing, T. & Lehr, T. A Physiologically‐Based Quantitative Systems Pharmacology Model of the Incretin Hormones GLP‐1 and GIP and the DPP4 Inhibitor Sitagliptin. *CPT Pharmacometrics Syst. Pharmacol.* psp4.12520 (2020).doi:[10.1002/psp4.12520](https://doi.org/10.1002/psp4.12520)

**Comment by SCZ1174** [comment]
Dear Mr Balazki,
(What a great study!)
I am a modeling noob and there are a few things I don't understand while studying this article that I would like to ask you about.
1. When describing the degradation process of incretin hormones, I found in the mobi file that there is a parameter "K_water" included in the formula, but there is no mention of this parameter in the supplementary file. What is the role of this parameter here?
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/86552551/fea2e101-e009-41ad-a224-37e844f965b8)
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/86552551/70ed3adf-ae27-4d78-9947-b31721ae5e30)

2. Why is the metabolism of GLP-1 (7-36)amide and (9-36)amide to unknown metabolites by NEP a linear process, whereas the degradation of GIP (1-42) to unknown metabolites by NEP is modeled as a saturation process? What is the basis for defining the type of degradation process here? (and other descriptions of degradation processes)

3. For incretin hormones, the large molecule module is used for modeling, while for sitagliptin, the small molecule module should be used. How to merge small molecule and large molecule in mobi's "Molecules" module? Should I use "Merge" or just select both in PK-sim? (If the latter, should I use the large molecule module or the small molecule module in the simulation?)

4. The R package "MoBiToolboxForR_8.0.15" for graphing cannot be installed in R4.0.0 or above, is there any alternative?

With best regards，
SCZ

**Comment by PavelBal** [comment]
Hi @SCZ1174 ,

sorry for the delayed answer.

1. `k_water` is the partitioning coefficient of the molecule in the specific container, so basically the free fraction of the hormones in the interstitial space. As we assume the fraction unbound in plasma as 100% for the hormones, the value of `k_water` is also 1.

2. DPP4-mediated degradation processes are modeled as saturable reactions as prior information about the `Km` value exists (in vitro measured). For processes that we do not have any prior information, as in case of NEP mediated degradation (we are using NET as a surrogate for the "residual" degradation and do not know exactly what this process is), we usually assume linear kinetics and only add saturability if the data suggests it. In case of GIP (1-42), it was necessary to make the process saturable and fit the `Km` value in order to describe the data.
3. The "basic" PBPK model was created in PK-Sim and combined all molecules that are used in the final model. For this, a large molecules model was selected in PK-Sim, which allows simulation of both large as well as small molecules.
4. Unfortunately, the support of this R package has been discontinued and there is no version of R4 available.

**Reply by SCZ1174** [reply]
Thank you very much for your reply!
For the third point, do you mean that the macromolecule module in PK-sim can simulate both macromolecules and small molecules? (Can small molecules be simulated without checking the "Is small molecule" option?)

**Comment by PavelBal** [comment]
You can select the "Model for large molecules" and combine macromolecules (with the box "Is small molecule" unchecked) with small molecules (with the box "Is small molecule" checked) in one simulation.

**Reply by SCZ1174** [reply]
It's clear to me now, thank you so much!
